src/app/events/[slug]/page.tsx

```diff
@@ -64,30 +73,36 @@ export default function EventEntry({ params }: EventProps) {
     <PageLayout>
       <StructuredDataScript structuredData={generateStructuredData(data)} />
       <div className="grid gap-4">
-        <TagLabel borderColor="brand-100">
-          {getCategoryLabel({ collectionName: 'event_entries', category })}
-        </TagLabel>
+        <TagGroup
+          label={[
+            getCategoryLabel({ collectionName: 'event_entries', category }),
+            eventHasConcluded ? 'Concluded Event' : undefined,
+          ]}
+        />
         <PageHeader
```

Comment:
Would be nice to add `objectFit: 'cover'` to the `PageHeader`. I thought I did it as part of the pervious PR.

Reply:
Ah! Nice catch! I was wanting to ask about this! 🙌🏼